org.upnproutercontrol.UPnPRouterControl.json

```diff
@@ -0,0 +1,75 @@
+{
+   "app-id":"org.upnproutercontrol.UPnPRouterControl",
+   "branch":"unstable",
+   "runtime":"org.gnome.Platform",
+   "runtime-version":"3.38",
+   "sdk":"org.gnome.Sdk",
+   "command":"upnp-router-control",
+   "rename-desktop-file":"upnp-router-control.desktop",
+   "finish-args":[
+      "--share=network",
+      "--socket=session-bus",
```

Comment:
what do you need this permission for? (I can guess but I'd like to hear from you, and to understand if it's possible to scope more narrowly)

Reply:
I need to use it to communicate with the router in the local network. The network operations are performed by libgupnp and libgssdp. The UPnP protocol uses HTTP and SOAP technologies on top of IP.